controller/channel.go

```
@@ -522,14 +523,14 @@ func validateChannel(channel *model.Channel, isAdd bool) error {
 			if !strings.HasPrefix(trimmedKey, "{") {
 				return fmt.Errorf("Codex key must be a valid JSON object")
 			}
-			var keyMap map[string]any
-			if err := common.Unmarshal([]byte(trimmedKey), &keyMap); err != nil {
+			oauthKey, err := codex.ParseOAuthKey(trimmedKey)
+			if err != nil {
 				return fmt.Errorf("Codex key must be a valid JSON object")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)